src/Raven.Server/Documents/Indexes/IndexMerging/IndexMerger.cs

```diff
@@ -352,7 +352,6 @@ private static void MergeSelectExpressionsAndFields(MergeProposal mergeProposal,
 
                 mergeSuggestion.CanMerge.Add(curProposedData.IndexName);
                 DataDictionaryMerge(mergeSuggestion.MergedIndex.Fields, curProposedData.Index.Fields);
-                mergeSuggestion.MergedIndex.Fields = mergeSuggestion.MergedIndex.Fields;
```

Comment:
Why remove this?

Reply:
Must have been Resharper. It is not useful though, so it shouldn't do any harm